projects/storefrontapp-e2e-cypress/cypress/helpers/customer-ticketing/customer-ticketing.ts

```diff
@@ -100,7 +101,8 @@ export function clickSubmit(){
 }
 
 export function verifyRequestCompleted(){
-  cy.get('cx-global-message').contains('Request Created');
+  cy.get('cx-global-message').contains('Request created.');
+  cy.get('cx-global-message').contains('Request created.').should('not.exist', { timeout: 10000 })
 }
```

Comment:
We need to make it more generic. Something that allows us to validate the X message is shown.
Maybe pass the text as a parameter.
I can see you are testing that the message is shown and shortly after is gone. ?